testing/py3-subunit/APKBUILD

```diff
@@ -0,0 +1,25 @@
+# Contributor: Robert Sacks <robert@sacks.email>
+# Maintainer: Robert Sacks <robert@sacks.email>
+pkgname="py3-subunit"
+_pyname="python-subunit"
+pkgver="1.3.0"
+pkgrel=0
+pkgdesc="Python implementation of subunit test streaming protocol"
+url="https://launchpad.net/subunit"
+arch="noarch"
+license="Apache-2.0 BSD-3-Clause"
+options="!check" # Test suite issues
+depends="python3 py3-extras py3-testtools"
+makedepends="py3-setuptools py3-fixtures py3-hypothesis py3-testscenarios"
```

Comment:
Please check what of this can be moved to `checkdepends`

Reply:
Fixed